src/dosbox.cpp

```diff
@@ -434,13 +440,19 @@ void DOSBOX_Init()
 	constexpr auto changeable_at_runtime = true;
 
 	/* Setup all the different modules making up DOSBox */
-	const char *machines[] = {"hercules",      "cga",
-	                          "cga_mono",      "tandy",
-	                          "pcjr",          "ega",
-	                          "vgaonly",       "svga_s3",
-	                          "svga_et3000",   "svga_et4000",
-	                          "svga_paradise", "vesa_nolfb",
-	                          "vesa_oldvbe",   0};
+	const char* machines[] = {"hercules",
+	                          "cga",
+	                          "cga_mono",
+	                          "tandy",
+	                          "pcjr",
+	                          "ega",
+	                          "svga_s3",
+	                          "svga_et3000",
+	                          "svga_et4000",
+	                          "svga_paradise",
+	                          "vesa_nolfb",
+	                          "vesa_oldvbe",
+	                          0};
```

Comment:
I assume adding a plain `vga` option is rather pointless as all `svga_*` types are basically that; pre-VESA VGA cards with extra "Super VGA" features.